src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -96,14 +88,16 @@ const getMessageFromBraze = async (
 	const appboyTiming = initPerf('braze-appboy');
 	appboyTiming.start();
 
-	appboy.initialize(apiKey, SDK_OPTIONS);
+	const brazeMessages = new BrazeMessages(appboy);
 
-	const canShowPromise: Promise<CanShowResult> = new Promise((resolve) => {
-		let subscriptionId: string | undefined;
+	const messages = brazeMessages.getMessagesForBanner();
```

Comment:
Given that this method returns a promise which _may_ resolve with a single message what do you think about renaming to `getMessageForBanner`, singular? (Sorry I think this may have been my naming in the first place!)

Reply:
Yes, that makes perfect sense - all updated now.